modules/cnf-programming-the-sriov-fec-operator-acc100.adoc

```diff
@@ -0,0 +1,345 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="configuring-the-sr-iov-fec-operator-intel-vran-dedicated-accelerator-acc100_{context}"]
+= Configuring the SR-IOV-FEC Operator for the Intel® vRAN Dedicated Accelerator ACC100
+
+This section describes how to program the Intel® vRAN Dedicated Accelerator ACC100.
+The Intel® vRAN Dedicated Accelerator ACC100 accelerates 4G and 5G Virtualized Radio Access Networks (vRAN) workloads increasing the overall compute capacity of a commercial, off-the-shelf platform.
+This device is also known as Mount Bryce.
+
+The SR-IOV-FEC Operator handles the management of the Forward Error Correction (FEC) devices used to accelerate the FEC process in vRAN L1 applications.
+
+Configuring the SR-IOV-FEC Operator involves:
+
+* Creating the desired virtual functions (VFs) for the FEC device
+* Binding the VFs to the appropriate drivers
+* Configuring the VF queues for desired functionality in a 4G or 5G deployment
++
+[NOTE]
+====
+The role of Forward Error Correction (FEC) is to correct transmission errors, where certain bits in a message can be lost or garbled. Messages can be lost or garbled due to noise in the transmission media, interference, or low signal strength.
+Without FEC, a garbled message would have to be resent, adding to the network load and impacting throughput and latency.
+====
+
+.Prerequisites
+
+* Intel® FPGA ACC100 5G/4G card
+* Node or nodes installed with the OpenNESS Operator for Wireless FEC Accelerators
+* RT kernel configured with Performance Addon Operator
+* Log in as a user with `cluster-admin` privileges
++
+[NOTE]
+====
+All the commands run in the `vran-acceleration-operators` namespace.
+====
+
+.Procedure
+
+. Verify that the SR-IOV-FEC Operator is installed and that pods are running:
```

Comment:
Snip it because the next step checks the pods.

"Verify that the SR-IOV-FEC Operator is installed:"